### PR TITLE
[Hotfix] Final fix LGTM alerts

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Final correction of alerts found by the LGTM, one of which is marked as false-positive

Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

I hope that after applying this PR, the LGTM`s check will be successful.

I think it would be nice to release v2.0.0 without LGTM alerts.

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c [hunk 1]
```diff
@@ -1226,7 +1226,7 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
     results = g_array_sized_new (FALSE, TRUE, sizeof (detectedObject), 100);
 
     boxes = &input[0];
-    if (num_tensors >= MOBILENET_SSD_MAX_TENSORS)
+    if (num_tensors >= MOBILENET_SSD_MAX_TENSORS) // lgtm[cpp/constant-comparison]
```
Comment (wooksong) [inline ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:1229]:
```suggestion
    if (num_tensors >= MOBILENET_SSD_MAX_TENSORS) /* lgtm[cpp/constant-comparison] */
```

File: gst/nnstreamer/tensor_common.c [hunk 2]
```diff
@@ -1753,7 +1753,7 @@ gst_tensor_meta_info_convert (GstTensorMetaInfo * meta, GstTensorInfo * info)
 
   info->type = meta->type;
 
-  for (i = 0; i < NNS_TENSOR_META_RANK_LIMIT; i++) {
+  for (i = 0; i < NNS_TENSOR_META_RANK_LIMIT; i++) {    // lgtm[cpp/constant-comparison]
```
Comment (wooksong) [inline gst/nnstreamer/tensor_common.c:1756]:
```suggestion
  for (i = 0; i < NNS_TENSOR_META_RANK_LIMIT; i++) { /* lgtm[cpp/constant-comparison] */
```

File: ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c (hunk 1 above)
Comment (tdrozdovsky) [reply ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c]: Done!

File: gst/nnstreamer/tensor_common.c (hunk 2 above)
Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_common.c]: Done! Thank you!

**Review by jaeyun-jung — APPROVED** [review]

**Review by wooksong — APPROVED** [review]
LGTM!

**Pull request merged**